deny.toml

```diff
@@ -27,6 +27,11 @@ yanked = "warn"
 id = "RUSTSEC-2023-0071"
 reason = "the marvin attack only affects private key decryption, not public key signature verification"
 
+[[advisories.ignore]]
+id = "RUSTSEC-2024-0399"
+reason = "0.22.4 is unaffected https://github.com/rustsec/advisory-db/pull/2144"
+crate = "rustls@0.22.4"
```

Comment:
```
error[unexpected-keys]: found 1 unexpected keys, expected: ["id", "reason"]
   ┌─ /__w/neon/neon/deny.toml:33:1
   │
33 │ crate = "rustls@0.22.4"
   │ ─────
```

If we need a newer `cargo-deny` for that syntax, you can bump the version here:
https://github.com/neondatabase/neon/blob/6f7aeaa1c563906fe14a4f62870d9fcd62436e02/build-tools.Dockerfile#L266

Reply:
The patch to the advisory db was merged, so I'll get rid of this ignore